src/net.h

```diff
@@ -999,6 +1004,38 @@ class CConnman
     mapMsgTypeSize m_msgtype_bytes_recv;
     mapMsgTypeSize m_msgtype_bytes_sent;
 
+    struct NetMsgStatsKey {
+        // protocol.cpp has a list of allNetMessageTypes which use the NetMsgType namespace.
+        // Can I treat NetMsgType like an enum?
+        std::string msg_type;
+        ConnectionType conn_type;
+        Network net_type;
+
+        bool operator<(const NetMsgStatsKey& rhs) const
+        {
+            if (msg_type < rhs.msg_type) {
+                return true;
+            } else if (msg_type == rhs.msg_type) {
+                if (conn_type < rhs.conn_type) {
+                    return true;
+                } else if (conn_type == rhs.conn_type) {
+                    if (net_type < rhs.net_type) {
+                        return true;
+                    }
+                }
+            }
+            return false;
+        }
+    };
+
+    struct NetMsgStatsValue {
+        int msg_count;
+        uint64_t num_bytes;
```

Comment:
variable name suggestions, in theme with simplifying 
```suggestion
    struct MsgStatsValue {
        int count;
        uint64_t bytes;
```

Reply:
Similar to the previous comment about `MsgStatsKey` (formerly known as `MsgStatsValue`), I have updated this  name of this struct and dropped the "Net":  https://github.com/satsie/bitcoin/commit/734ac6fd6c3c3f5f9f395ef629e0fd8e2bc85edc :) 

I left `msg_count` and actually changed `bytes` to `byte_count`. 
- I think by itself, `count` could possibly be interpreted as the total number of stats being returned. 
- Without looking at the data type, `bytes` could mean the actual bytes themselves. 
Now I can't tell you how realistic it is for someone to misinterpret `count`, or how reasonable it is to make accommodations for readers that don't bother to look at data types. For the latter, I think I have a lot of conditioning from years of working in a dynamically typed language. I'm not sure what the standards are in C++ land, and am happy to drop the "count" off `byte_count` if it's considered acceptable. By that same line of thought, "count"/"msg_count" could then maybe be changed to just `messages` as the int data type would indicate we're not talking about the actual messages themselves.

Though.... as I look at all this I realize when I switch the data structure from a map to an array, none these names should matter. Earlier today I finally fixed a bug in my RPC code that was driving me bananas for days. I was looking for some small wins before diving back into the bigger stuff. I probably should have thought ahead before tackling this naming feedback :grimacing: